src/foam/nanos/jetty/HttpServer.js

```diff
@@ -469,9 +493,19 @@ foam.CLASS({
             keyStore.load(bais, this.getKeystorePassword().toCharArray());
           }
 
-          // 2. enable https
-          HttpConfiguration https = new HttpConfiguration();
-          https.addCustomizer(new SecureRequestCustomizer());
+          // Enable https
+          HttpConfiguration config = new HttpConfiguration();
+
+          SecureRequestCustomizer src = new SecureRequestCustomizer();
+          if ( getAllowSNIDisable() ) {
+            String hostname = System.getProperty("hostname");
+            if ( hostname != null &&
+                 hostname.indexOf(".") < 0 ) {
```

Comment:
Do we need this no "." in hostname check?
I ask because for intuit api integration i had to use a url for local that had a . (e.g envoice.localhost) for redirecturi

Reply:
Replaced with property on HttpServer so one can explicitly list hostnames to disable check on. 
disableSNIForHostnames: ['localhost']

Reply:
sorry to be a pain but i think this approach has the downside that everyone would have to modify HttpServer to add there theme based urls like envoice, nbp, intuit that they use for https://intuit:8443 for example (and any new theme  domain we add in the future)
maybe best approach is both, check for . and only check disableSNIForHostnames if there is a dot?

Reply:
Found the build layering didn't support this either, as -u  (deployment/resources/u) is applied last. 
Refactored to a boolean - 'disableSNIHostCheck', which defaults to 'true' - which is what everyone needs until production deployment.  Then a deployment/production/services.jrl can set it to 'false', if required.